### DISCUSSION TITLE
NC - No Chord

### DISCUSSION THREAD

**Discussion by andermikael** [jjazzboss/JJazzLab#307] [Q&A]
I have this sheet music and there is an intro for 3 bars. Then one bar N. C. (which I believe stands for No Chord).
Then the verse starts.

How can I input a No Chord bar?

I tried setting up a section but I get an error:
starting chord symbol missing for section B at bar 4

**Comment by jjazzboss** [comment]
There is no support for NC  No Chord. A workaround  is to create a chord symbol and edit it to make it "shot" with "extended instruments" (see  Interpretation tab in the chord symbol editor)
